tests/subsys/dfu/dfu_target_stream/storage.overlay

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: LicenseRef-Nordic-5-Clause */
```

Comment:
Can you add a comment explaining what this file is needed for?

Reply:
Removed the file as its no longer needed